corehq/blobs/tests/test_init.py

```diff
@@ -29,6 +29,7 @@ def test_get_blobdb(self, msg, root=True, blob_dir=None):
             temp_dir=None,
             blob_dir=blob_dir,
         )
+        mod._db = []  # reset singleton
```

Comment:
Why? `_db` is a private stack of contextually assigned blob dbs. This will discard any [context that is setup outside of this test](https://github.com/dimagi/commcare-hq/blob/0c79c9af08e231efe7d1892c57f26ee38f22ca9b/testrunner.py#L64), and could cause other tests to fail or write to the wrong blob db. What do you think of using something like [`testil.replattr`](https://github.com/millerdev/testil/blob/master/testil.py#L110-L115) to temporarily reset `mod._db` for the duration of this test?